code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -656,6 +657,10 @@ Code:
 			loc:mode = 433
 			mode = 433
 
+		if("Supply Orders")
+			loc:mode =47

```

Comment:
defines would be good

actually that code doesn't even need to be there, I was fiddling around with some shit before I realized it was ALREADY IN